NimbleMedium/Sources/Data/NetworkAPI/Models/APIUser.swift

```diff
@@ -2,16 +2,23 @@
 //  APIUser.swift
 //  NimbleMedium
 //
-//  Created by Minh Pham on 26/08/2021.
+//  Created by Minh Pham on 30/08/2021.
 //
 
 import Foundation
 
-struct APIUser: User, Decodable {
-
+struct APIUser: User, Codable {
```

Comment:
I would prefer to use only `Decodable`, currently, we don't need to encode the user object and send it to the server. We encode the user object to save it to Keychain so I suggest something like this:
```swift
struct KeychainUser: User, Codable { ... }
```

Reply:
@suho in fact, the `APIUser` and `KeychainUser` are basically the same. In my opinion, there is no need to create 2 different objects of duplicated fields.
Also, about this idea `we don't need to encode the user object and send it to the server`, we do have the task to update user data as well, so we will need it eventually.

Reply:
So what do you think if we update the name to `CodableUser`?
```suggestion
struct CodableUser: User, Codable {
```

Reply:
CodableUser sounds okay to me 👍